src/rideTask.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef __RIDE_HPP__
+#define __RIDE_HPP__
+#include "task.hpp"
+
+#include "Particle.h"
+#include "product.hpp"
+
+#define RIDE_RGB_LED_COLOR    RGB_COLOR_WHITE
+#define RIDE_RGB_LED_PATTERN_GPS  LED_PATTERN_BLINK
+#define RIDE_RGB_LED_PERIOD_GPS   500
+#define RIDE_RGB_LED_PATTERN_NOGPS  LED_PATTERN_SOLID
+#define RIDE_RGB_LED_PERIOD_NOGPS   0
+#define RIDE_RGB_LED_PRIORITY LED_PRIORITY_IMPORTANT
```

Comment:
These should probably be defined in `product.hpp`, since this is a behavioral configuration

Reply:
moved to `product.hpp` in ed4e634